Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -16,57 +16,70 @@ import groovy.cli.commons.*
- *                                       output directory
+ * -w,--workDir <dir>                             Absolute path to the DBB build
+ *                                                output directory
+ * -properties,--packagingPropertiesFile <file>   Absolute path of a property file
+ *                                                containing application specific
+ *                                                packaging details. 
+ *                                                                                                                                         
  * Optional:
- * -t,--tarFileName <filename>           Name of the package tar file.
- *                                       (Optional)
- * -d,--deployTypes <deployTypes>        Comma-seperated list of deployTypes
- *                                       to filter on the scope of the tar
- *                                       file. (Optional)
- * -verb,--verbose                       Flag to provide more log output.
- *                                       (Optional)
+ * -t,--tarFileName <filename>                    Name of the package tar file.
+ *                                                (Optional)
+ * -d,--deployTypes <deployTypes>                 Comma-seperated list of deployTypes
+ *                                                to filter on the scope of the tar
+ *                                                file. (Optional)
+ * -verb,--verbose                                Flag to provide more log output.
+ *                                                (Optional)
+ * -il,--includeLogs                              Comma-separated list of files/patterns
+ *                                                from the USS build workspace                                               
  *                                       
  * Optional Artifactory Upload opts:
  *
- * -p,--publish                          Flag to indicate package upload to
- *                                       the provided Artifactory server.
- *                                       (Optional)
- * -prop,--propertyFile <propertyFile>   Absolute path of a property file
- *                                       containing application specific
- *                                       Artifactory details. (Optional)
- * -v,--versionName <versionName>        Name of the Artifactory version.
- *                                       (Optional)
- * -h,--help                             Prints this message
+ * -p,--publish                                   Flag to indicate package upload to
+ *                                                the provided Artifactory server.
+ *                                                (Optional)
+ * -artifactory,
+ *   --artifactoryPropertiesFile <propertyFile>   Absolute path of a property file
+ *                                                containing application specific
```

Comment:
I'm wrong, or you are breaking the current behavior?
Can we keep the `-prop,--propertyFile <propertyFile>`with a deprecated notice?

Reply:
This is now implemented. Resolved